web-client/src/views/DocketRecord/DraftDocumentViewerDocument.jsx

```diff
@@ -115,6 +117,7 @@ export const DraftDocumentViewerDocument = connect(
                 <Button
                   link
                   icon="pencil-alt"
+                  id="remove-docket-entry"
```

Comment:
Nit: I believe this button is to remove the signature off the docket entry, not remove the docket entry entirely?

Reply:
change implemented.